pkg/universe.dagger.io/python/python.cue

```diff
@@ -42,7 +42,7 @@ import (
 
 	docker.#Run & {
 		command: {
-			name:   "python3"
+			name:   string | *"python3"
```

Comment:
The default should be `"python"`, instead of `"python3"`. Python users should pull from a python image (like in your test), rather than install manually with `alpine.#Build`. I'd reflect this in the tests as a documentation stand-in.

For this particular action of running a python script, I'd think it's ok not to set a default but there's two use cases:

1. If you're really installing manually, it's surprising to me why the `apk` package doesn't symlink `python`;
2. If you have an image with multiple python versions (I use it to run tox). I'd just use `docker.#Run` if that was the case, but someone could want to reuse that image so it's ok.

For other cases, it should be enough to update `PATH` in `env` (image config, or locally in action).

Reply:
Agreed on all points, but as you call out `apk add python3` does not symlink python. I expect to make a new PR today to change the image to an official Python image rather than building one on Alpine. Was just giving the Discord some chance to weigh in in case there was a philosophical concern I wasn't aware of.